library-config/src/lib.rs

```diff
@@ -240,6 +244,27 @@ impl LibraryConfigName {
     }
 }
 
+#[repr(C)]
+#[derive(Clone, Copy, serde::Deserialize, Debug, PartialEq, Eq, Hash)]
+#[serde(rename_all = "SCREAMING_SNAKE_CASE")]
+#[allow(clippy::enum_variant_names)]
+pub enum LibraryConfigSource {
+    // Order matters, as it is used to determine the priority of the source.
+    //  The higher the value, the higher the priority.
+    LocalFile = 0,
+    Managed = 1,
+}
+
+impl LibraryConfigSource {
+    pub fn to_str(&self) -> &'static str {
+        use LibraryConfigSource::*;
+        match self {
+            LocalFile => "local_file",
+            Managed => "fleet_automation",
+        }
+    }
+}
```

Comment:
This is not used anywhere?

Reply:
It's used in client libraries, exactly like `LibraryConfigName`